### PR TITLE
Check BindExpression callee for reference - fixes T6984

### PR DESCRIPTION
**Pull request by loganfsmyth**
We weren't properly checking `callee` on `BindExpression`.

### CODE REVIEW

**Comment by hzoo** [comment]
https://phabricator.babeljs.io/T6984. Should we add an exec.js test for this one?

**Comment by loganfsmyth** [comment]
@hzoo Your call, I don't feel strongly one way or the other.

**Comment by hzoo** [comment]
I think since it was an issue found after running it we could do an exec.js test

File: packages/babel-plugin-transform-function-bind/test/fixtures/regression/T6984/options.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["transform-function-bind", "transform-es2015-classes"]
```
Comment (hzoo) [inline packages/babel-plugin-transform-function-bind/test/fixtures/regression/T6984/options.json]: Not a big deal, but do we want to use external helpers to make it shorter?

**Pull request merged**